alpine/entry.sh

```diff
@@ -24,15 +24,34 @@ mount --move /tmp /dev
 ln -sf /dev/pts/ptmx /dev/ptmx
 
 mount -t debugfs nodev /sys/kernel/debug
-udevd & 
-udevadm trigger &> /dev/null
+
+if [ "$INITSYSTEM" = "on" ]; then
+	GREEN='\033[0;32m'
+	echo -e "${GREEN}OpenRC init system enabled."
+	env >> /etc/docker.env
+	printf '#!/bin/bash\n' > /etc/resinApp.sh
+	printf '%q ' "$@" >> /etc/resinApp.sh
+	chmod +x /etc/resinApp.sh
+
+	touch /var/log/resinapp.log
+	tail -f /var/log/resinapp.log &
```

Comment:
This log file will generate SD cards writes for the logs and will grow indefinitely